### PR TITLE
replace module separator with path separator

### PR DESCRIPTION
**Pull request by Czaki**
Fix bug reported on zulip https://napari.zulipchat.com/#narrow/stream/309872-plugins/topic/Plugins.20showing.20up.20in.20napari.20plugin.20dialogue/near/411169527

### CODE REVIEW

File: src/npe2/manifest/_bases.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def from_file(cls, path: Union[Path, str]):
             try:
                 import tomllib
             except ImportError:
-                import tomli as tomllib  # type: ignore [no-redef]
+                import tomli as tomllib
```
Comment (Czaki) [inline src/npe2/manifest/_bases.py:82]:
```suggestion
                import tomli as tomllib # type: ignore [no-redef]
```

**Review by brisvag — APPROVED** [review]

**Review by DragaDoncila — APPROVED** [review]
Thanks @Czaki! I think this is fine to merge without napari tests passing

**Comment by nclack** [comment]
@DragaDoncila @Czaki What do you all think about making a release to get this fix out to pypi?

**Comment by Czaki** [comment]
I think that it is a good idea. The only thing is that I would like to check if new release do not break 0.4.18. It requires half hour for check.

**Pull request merged**